### ISSUE TITLE
Rename gesture

### ISSUE THREAD

**Issue by alexopalka** [roboleary/LeapTrainer.js#4]
They this would probably be a super quick fix, it would be beneficial for users to be able to just straight up change the name of their gesture from the overlay.

**Comment by roboleary** [comment]
Yep, you're right - and deletion of gestures could be an idea too, for completeness.  

I'll add this to the v0.3 release - thanks!
